specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.3-preview/keys.json

```diff
@@ -866,6 +866,64 @@
         }
       }
     },
+    "/keys/{key-name}/{key-version}/release": {
+      "post": {
+        "tags": [
+          "Keys"
+        ],
+        "operationId": "releaseKey",
+        "summary": "Releases a key.",
+        "description": "The release key operation is applicable to all key types. The target key must be marked exportable. This operation requires the keys/release permission.",
```

Comment:
Should this differentiate between export and release? To be honest, I'm not entirely sure on the difference either and I'm sure customers will raise the question later.

Reply:
Export is the reverse of Import. Release is a specialized Export used with Trusted Execution Environments. They are separated to enable more granular access control as Export must be closely controlled.

Reply:
That's what I thought :) but I would put something to that effect in the description. I anticipate customers will have this same question, and these descriptions should also end up being the default doc comments for the SDK code.